runtime/lib/ttnn/runtime.cpp

```diff
@@ -50,6 +50,11 @@ Tensor createTensor(std::shared_ptr<void> data,
   return Tensor(tensor, data, DeviceRuntime::TTNN);
 }
 
+tt::target::DataType getTensorDataType(Tensor tensor) {
+  ::ttnn::Tensor nnTensor = tensor.as<::ttnn::Tensor>(DeviceRuntime::TTNN);
```

Comment:
Thanks for the update. Can we make this a const reference just so we don't copy the ttnn tensor, which can be costly?

Reply:
updated as suggested.